lib/mito/utilities/SharedPointer.h

```diff
@@ -11,7 +11,7 @@ namespace mito::utilities {
 
     // declaration
     template <class Resource>
-    requires ReferenceCountedObject<Resource>
+    // requires ReferenceCountedObject<Resource>
```

Comment:
remove commented code if not needed

Reply:
The {requires} statement has now been reinstated.